tracer/tracer.go

```diff
@@ -133,6 +136,42 @@ func (t *Tracer) SetServiceInfo(name, app, appType string) {
 	}
 }
 
+// SetMeta adds an arbitrary meta field at the tracer level.
+// This will append those tags to each span created by the tracer.
+func (t *Tracer) SetMeta(key, value string) {
+	if t == nil { // Defensive, span could be initialized with nil tracer
+		return
+	}
+
+	t.metaMu.Lock()
+	if t.meta == nil {
+		t.meta = make(map[string]string)
+	}
+	t.meta[key] = value
+	t.metaMu.Unlock()
+}
+
+// getAllMeta returns all the meta set by this tracer.
+// In most cases, it is nil.
+func (t *Tracer) getAllMeta() map[string]string {
```

Comment:
Any reason to implement this one? We are dealing without it in Python and Ruby, so I'd vote to keep the API simple if we can.